docs/sources/menu.yaml

```
@@ -395,6 +395,8 @@
   children:
     - name: Overview
       link: /enterprise/
+    - name: Activate license
+      link: /enterprise/activate-license/     
```

Removing some trailing whitespace:
```suggestion
    - name: Activate license
      link: /enterprise/activate-license/
```